src/eval_suite.py

```diff
@@ -1,7 +1,7 @@
 import json
 import lm_eval
 from models import LongNetModelHF, RetNetModelHF, TransformerModelHF
-from torchscale.architecture.config import RetNetConfig, DecoderConfig
+from torchscale.architecture.config import RetNetConfig, DecoderConfig, LongNetConfig
```

Comment:
If you have a second, it would be nice to have this list of imports alphabetized like the import on the line above.

Reply:
Fixed in 586c44c.